src/Command/Diff.php

```diff
@@ -86,25 +86,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $handler = new DiffHandler($envName, $cwd, $output->isDecorated());
+        $errorIo = $this->getErrIo($input, $output);
```

Comment:
But... that's [your code](https://github.com/manala/manalize/commit/ef3dfb68dbc0c2781208b144a164cb5e40f76b57#diff-45cbdbae2d21a8c8fae7d15a87e1b135R99) 😅